PYME/Acquire/Hardware/Camera.py

```diff
@@ -616,6 +617,11 @@ def GetAcquisitionMode(self):
                 MODE_HARDWARE_START_TRIGGER
                     camera waits for an external hardware trigger and enters 
                     free-running internal timing mode once received
+                MODE_HARDWARE_TRIGGER_SYNC
+                    (stops then) starts exposing a frame every time an
+                    external hardware trigger is received by the camera.
+                    Allows timing the camera frames entirely off an external
+                    clock.

```

Comment:
A little unclear exactly how this works. Does the exposure start on the rising edge and end on the falling edge? Or end on the next rising edge? 

Does the next exposure start on a rising or falling edge? 

Looking at the docstrings associated with the changes to the orca class, it seems like it might start the next exposure on the falling edge. If so, that should be documented here. If this is the case, it might also make sense to call this, e.g. `MODE_HARDWARE_TRIGGER_CHOPPER` or similar, as this is likely to be one of the few/only scenarios where it makes sense to be triggering at twice the trigger frequency (a normal frame clock would typically be as described in the first sentence  of this comment).